### ISSUE TITLE
Retailer list not up to date

### ISSUE THREAD

**Issue by ba32107** [BeauAgst/avios-reminder#8]
Hi! Great extension. I have a bit of a problem though - not all retailers show up in the list. I think the API you are using does not return all the official ones.

Examples that don't show up, but present on the BA shopping portal:
* Decathlon
* UberEats
* Booking.com
* Etsy

These are pretty important ones. Could you look into if there is maybe a different API to use?

**Comment by BeauAgst** [comment]
Hey @ba32107 - this list is coming directly from BA themselves. [(see here)](https://github.com/BeauAgst/avios-reminder/blob/master/src/utils/retailers.ts#L4).

I've just had another check, and this URL hasn't changed. The examples you gave unfortunately don't appear in their own extension either.

The API used by shopping.ba.com has all the data we need, but because there are no domain data in the response it would be impossible for us to determine whether a popup should be shown or not. If you [see here](https://github.com/BeauAgst/avios-reminder/blob/master/src/background.ts#L3), we are checking the URL of the page against the URL in our JSON array.

If you have any suggestions, let me know!

**Comment by ba32107** [comment]
Hi @BeauAgst, hmm I see, this is tough. I had a look at the data that shopping.ba.com returns and indeed it's incomplete.

Out of curiosity, where did you get the URL that you're currently using? Did you extract it from their extension?

**Comment by BeauAgst** [comment]
Indeed, I found it snooping through their [official extension](https://chrome.google.com/webstore/detail/british-airways-avios-col/hbhpmkcbobfmndllbcllnooeaghkdoka/)

**Comment by BeauAgst** [comment]
Hey @ba32107, it looks like the API endpoint has been decommissioned and so I have updated it to the new one and submitted it for review in the store. It may be possible that the list is now an accurate reflection of the BA Shopping portal. Let me know!
